### PR TITLE
fixed according Pavlo's comments

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,8 +289,9 @@ Each response from the AbuseIPDB API for the supported observables generates
 the following CTIM entities:
 
 - `Verdict` is based on `.data.abuseConfidenceScore`
-- `Judgement` is based on `.data.reports[].`
-- `Sighting` is based on `.data.reports[].comment`
-- `Indicator` is based on `.data.reports[].categories[]`
+- `Judgement` is based on each report in `.data.reports[].`
+- `Sighting` is based on each report comment `.data.reports[].comment`
```
Comment (oshtaier) [inline README.md:293]: report comment in `.data.reports[].comment`?

Comment (mistergreen22) [reply README.md:293]: hmm can be change to
 "comment to report in `.data.reports[].comment`" 
but I think current case also valid, with small correction "report comment's in `.data.reports[].comment`"

Comment (mistergreen22) [reply README.md:293]: ou i get what you mean, that "in" missed

Comment (GeorgiyII) [reply README.md]: done

**Pull request merged**